File: core/feature54_test.go
package main

import "testing"

var (
	feature54Names      = []string{"SR", "BR", "KS", "GS", "PointsL"}
	feature54Accuracies = []string{"%.0f", "%.0f", "%.1f", "%.1f", "%.0f"}
	feature54Types      = []bool{true, true, true, true, false, false, false}
)

func Test01feature54(t *testing.T) {
	arguments := map[string]interface{}{"K": 200, "G": 300, "L": 200, "A": 60, "N": 350, "M": 400, "P": 200, "V": 350, "T": 6}
	expected := map[string]interface{}{"SR": 412, "BR": 618, "KS": 104.7, "GS": 157.1, "PointsL": []interface{}{-12, -16, -58, -88}}
	genericTester(t, feature54(arguments), expected, feature54Names, feature54Accuracies, feature54Types)
}

File: core/feature54_test.js
var feature54Names = ["SR", "BR", "KS", "GS", "PointsL"];
var feature54Accuracies = [0, 0, 1, 1, 0];
var feature54Types = [true, true, true, true, false, false, false];

function test01feature54() {
  var arguments = {"K": 200, "G": 300, "L": 200, "A": 60, "N": 350, "M": 400, "P": 200, "V": 350, "T": 6};
  var expected = {"SR": 412, "BR": 618, "KS": 104.7, "GS": 157.1, "PointsL": [-12, -16, -58, -88]};
  genericTester(feature54_core(arguments), expected, feature54Names, feature54Accuracies, feature54Types);
}

test01feature54();
